filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobFileSystemManager.scala

```diff
@@ -57,10 +58,13 @@ case class BlobFileSystemManager(
       shouldReopenFilesystem match {
         case false => fileSystemAPI.getFileSystem(uri).recoverWith {
           // If no filesystem already exists, this will create a new connection, with the provided configs
-          case _: FileSystemNotFoundException => blobTokenGenerator.generateAccessToken.flatMap(generateFilesystem(uri, container, _))
+          case _: FileSystemNotFoundException =>
+            logger.debug(s"Creating new blob filesystem for URI $uri")
+            blobTokenGenerator.generateAccessToken.flatMap(generateFilesystem(uri, container, _))
         }
         // If the token has expired, OR there is no token record, try to close the FS and regenerate
         case true =>
+          logger.debug(s"Closing & regenerating token for existing blob filesystem at URI $uri")
```

Comment:
Debug is not enabled in Terra prod. I deliberately but perhaps misguidedly elected not to increase Control Plane Cromwell's log volume, with the thought that we would enable a different log level for CaaA.

Reply:
I'm guessing we would not enable DEBUG for user-plane Cromwell, but I'm also not really aware of what we typically log as DEBUG.

In any case, it shouldn't be a concern for the control-plane instance, since it will never use the BlobFilesystem.

Reply:
Great point, I will make it `INFO`. Thanks!